### DISCUSSION TITLE
A few questions about the currently returned user profile

### DISCUSSION THREAD

**Discussion by AGMETEOR** [stacklok/minder#2038] [General]
Merry xmas everyone! I was just looking at this user profile response and I have a few questions
```
{
"user": {
"id": 0,
"organizationId": "string",
"identitySubject": "string",
"createdAt": "2019-08-24T14:15:22Z",
"updatedAt": "2019-08-24T14:15:22Z"
},
"projects": [
{
"projectId": "string",
"name": "string",
"description": "string",
"createdAt": "2019-08-24T14:15:22Z",
"updatedAt": "2019-08-24T14:15:22Z"
}
]
}
```
- What is the purpose of `organizationId` field? Is it for some kind of SAML signup in the future? Or organizations can buy a stacklok license/subscription and add users?
- Should a project also include a provider field? Or a single project can have multiple providers?

**Comment by JAORMX** [comment]
Hey! So, initially, we had orgs and groups for entities in Minder. We eventually converted into using projects instead. Orgs were kept just in case but we're opting for removing them soon.

In terms of providers, we're thinking one should be able to specify multiple providers in a single project. We haven't fully fleshed out the providers API yet, but it'll come.
